src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -56,18 +57,22 @@ private static void setupCommandTests(Application application) {
      */
     private static void setupDrivers(Application application) {
         Job2dDriver loggerDriver = new LoggerDriver(false);
-        DriverFeature.addDriver("Simple Logger driver", loggerDriver);
+        UsageMonitorDriver loggerMonitor = new UsageMonitorDriver(loggerDriver);
```

Comment:
Do not use usage monitoring with every driver. Maybe, add 2 new drivers and indicate in a driver name, e.g. "Line Simulator with usage monitoring"